docs/product-manuals/cloud-console/troubleshooting/feedback-and-support.md

```diff
@@ -3,8 +3,10 @@ id: feedback-and-support
 title: Feedback and support
 ---
 
-If you have any problems, questions or suggestions you can contact us via the Cloud Console Feedback Form.
+If you have any problems, questions, or suggestions, contact us via the Cloud Console Feedback Form.
 
-**Feebback and support** can be submitted or requested via the corresponding entry in the navigation menu. If you have a **license agreement** with us you will be redirected to the [Support queue](https://jira.camunda.com/projects/SUPPORT/) at Camunda. In the Free Plan, contact can be made using the internal form:
+[//]:# (Can we provide a link to the feedback form here, or is it only accessed within Cloud Console itself via the Feedback and Support tab here?)
```

Comment:
In the beginning it was a built-in form. Now we just link to the Forum: https://forum.camunda.io/ - so we can use the forum link here.